.gitignore

```diff
@@ -127,6 +127,9 @@ bin/
 *.svclog
 *.scc
 
+# BinLog artifacts
+msbuild.*.ProjectImports.zip
```

Comment:
Just curious, is an external project imports zip now the default?

Reply:
I think this is the file that is created while the build is being run, but doesn't persist afterwards.  But if you look at the unstaged changes while a binlogged build is running you would see this file.